src/Raven.Client/Documents/Operations/MaintenanceOperationExecutor.cs

```diff
@@ -92,7 +92,8 @@ public async Task<Operation<TResult>> SendAsync<TResult>(IMaintenanceOperation<O
                 var command = operation.GetCommand(RequestExecutor.Conventions, context);
 
                 await RequestExecutor.ExecuteAsync(command, context, sessionInfo: null, token: token).ConfigureAwait(false);
-                return new Operation<TResult>(RequestExecutor, () => _store.Changes(_databaseName), RequestExecutor.Conventions, command.Result.Result, command.Result.OperationId, command.SelectedNodeTag ?? command.Result.OperationNodeTag);
+                var node = command.SelectedNodeTag ?? command.Result.OperationNodeTag;
```

Comment:
when `command.Result.OperationNodeTag` can be different from `command.SelectedNodeTag`?

Reply:
@lastav5 ?

Reply:
I've talked to stav about this. 
This is not a new code and It seems to be historically like this, we don't see how it can be different, but decided to keep it this way to avoid even more changes.